Tests/ClientTests/OnboardingTests/NimbusOnboardingFeatureLayerDefaultsTests.swift

```diff
@@ -0,0 +1,136 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+//
+// Tests are disabled as there's curretly a string issue. Will be enabled once
```

Comment:
nit: Can we add a ticket or something to track so we remember to re-enable the test in the future

Reply:
I would also argue it's preferable to disable tests using Xcode "Disable". When we run the test suite it tells us what test is disabled, and make it easier to find than commented code in tests.

![Screenshot 2023-05-10 at 1 48 30 PM](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/677c010f-8b9d-4e7e-9af3-f5c820e3b31a)

Reply:
For sure. I've [created it](https://mozilla-hub.atlassian.net/browse/FXIOS-6383), assigned it to me and it's in this sprint. I won't add it to the PR because build is green already.